drivechain-server/drivechain/script.go

```diff
@@ -9,15 +9,22 @@ import (
 // is for.
 // This implements BIP300-M5: https://en.bitcoin.it/wiki/BIP_0300
 func ScriptSidechainDeposit(slot uint8) ([]byte, error) {
+	// https://github.com/LayerTwo-Labs/bip300301_wallet/blob/f2177c32f7854980e43df242f0634a01be3618bf/src/wallet.rs#L695-L700
 	const OP_DRIVECHAIN = txscript.OP_NOP5
-	return txscript.NewScriptBuilder().AddOp(OP_DRIVECHAIN).AddData([]byte{slot}).Script()
+	return txscript.NewScriptBuilder().
+		AddOps([]byte{OP_DRIVECHAIN, txscript.OP_DATA_1}).
+		AddData([]byte{slot}).
+		AddOp(txscript.OP_TRUE).
+		Script()
```

Comment:
jsut a heads up: This looks like it's different than what's in the mainchain repo:

https://github.com/LayerTwo-Labs/mainchain/blob/c20ad4189aa8ebabc2dfcc98dcaf30c6cfab8c0c/src/sidechaindb.cpp#L450-L461

Reply:
Hm, old mainchain and new CUSF wallet has different scripts?